app/templates/include/subscribe_form.html

```diff
@@ -2,7 +2,7 @@
 {% settings_value "STATIC_KEYS_TEMPLATES" as static_values %}
 
 {% if static_values.subscribe_url %}
-    Subscribe here to be the first to know when are we open applications for next edition.
+    <p>Subscribe here to be the first to know when are we open applications for next edition.</p>
```

Comment:
Subscribe here to be the first to know when `applications for the next edition are open`.